lib/biran/erb_config.rb

```diff
@@ -29,6 +32,7 @@ def build_erb_env
       proc do
         @environment = config[:env]
         @app_config  = config
+        @config_index =  config_index || DEFAULT_CONFIG_INDEX
```

Comment:
Remind me if this prco scope is the same of the class instance... if so, this can be done as `@config_index ||= DEFAULT_CONFIG_INDEX`, being an `attr_accessor` it will be set as an ivar ont he instance.

Reply:
I'm not sure how to answer that... sorry. This value is changed each time the `save!` method is called. See https://github.com/amco/biran/pull/63/files#diff-213904ac166ac8b8343fa4c1ab0dece8R39 for reference. So I think of it as an instance variable on the template being processed at that moment and less on the whole class, since it is changing.
So we create an instance of the template file contents that we load
We want to reuse those contents to create multiple output files
So we pass in the config index for each of those output files desired along with an output file name.
So if `config_index` is the same as `@config_index` and I don't need to use the accessor to "load" a new value after setting it in the loop using the instance, then we probably can switch. I used this here like this because all the other values are referenced in this way. 

Plus in this context we are setting the `@config_index` instance variable for use in the template processing so this instance variable is only available in the template and not in the class... while the class value is used to set this all up. I guess maybe it looked weird doing `@config_index = @config_index || DEFAULT_CONIG_INDEX` where both of those `@config_index` refer to different things are not the same.

Reply:
> I guess maybe it looked weird doing @config_index = @config_index || DEFAULT_CONIG_INDEX where both of those @config_index refer to different things are not the same.

Yes this is the part that is confusing, and not sure if there might be actual bugs. So, maybe rename the `@config_index` set in here for `@template_config_index`?

Reply:
I don't like `@template_config_index` too long and when using it in the template seems redundant.  Maybe just have the binding instance just be `index` or `index_number`
I just like config because it was clear what the index was for. Maybe I will rename the class variable... let me think on it.

Reply:
Or, I see... if I name the class variable to `template_config_index` that does not look bad. I was thinking on the erb template side
So, this wouldn't be so bad I think...
```
@config_index = @template_config_index || DEFAULT_CONFIG_INDEX
```